install/jaxws/docs/JAXWS3.0-ReleaseNotes.html

```diff
@@ -50,7 +50,7 @@ <h2><a name="kit_contents">Kit Contents</a></h2>
    <li><p><strong>Jakarta XML Web Services TCK Signature, API, End-to-end, WS-I Conformance, Java-to-WSDL/WSDL-to-Java Mapping and Web Services Addressing Conformance tests:</strong></p>
    <ul>
       <li>A <strong>Signature test</strong> that checks that all of the public APIs are supported/defined as specified in the Jakarta XML Web Services Version 3.0 implementation that is being tested.</li>
-      <li><strong>API tests</strong> to verify the functionality of all the public APIs under <code>jakarta.xml.ws</code>, <code>jakarta.xml.ws.handler</code>, <code>jakarta.xml.ws.handler.soap</code>, <code>jakarta.xml.ws.http</code>, <code>jakarta.xml.ws.spi</code>, <code>jakarta.xml.ws.spi.http</code>, <code>jakarta.xml.ws.soap</code>,  <code>jakarta.xml.ws.wsaddressing</code>, <code>jakarta.jws</code>, and <code>jakarta.jws.soap</code>.</li>
+      <li><strong>API tests</strong> to verify the functionality of all the public APIs under <code>jakarta.xml.ws</code>, <code>jakarta.xml.ws.handler</code>, <code>jakarta.xml.ws.handler.soap</code>, <code>jakarta.xml.ws.http</code>, <code>jakarta.xml.ws.spi</code>, <code>jakarta.xml.ws.spi.http</code>, <code>jakarta.xml.ws.soap</code>,  <code>jakarta.xml.ws.wsaddressing</code>, <code>jakarta.jws</code> and <code>jakarta.jws.soap</code>.</li>
```

Comment:
this change reminds me there is a list of packages somewhere in the guide (section 3 or 4 I think) which is missing `jakarta.jws.*` packages. Can you add them there so the list is consistent, please?

Reply:
It's about 1.2.2 and 2.5.